utils/resultPrint.js

```diff
@@ -0,0 +1,36 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES, RESULT, RULES } from "../constants/messages.js";
+
+/**
+ * strike개수와 ball 개수 파악하여 값 출력 함수
+ * @param {strike와 ball 개수가 담긴 배열} result
+ * @returns
+ */
+const resultPrint = (result) => {
+  let strike = result[0];
+  let ball = result[1];
```

Comment:
```suggestion
  const strike = result[0];
  const ball = result[1];
```
바뀌지 않는 것 같아서 `const`로 선언해도 좋을 것 같아요~!